tensorflow/python/data/ops/dataset_ops.py

```diff
@@ -36,13 +36,14 @@
 from tensorflow.python.compat import compat
 from tensorflow.python.data.ops import iterator_ops
 from tensorflow.python.data.ops import options as options_lib
+from tensorflow.python.data.ops import common
+from tensorflow.python.data.ops.common import StructuredFunctionWrapper
+from tensorflow.python.data.ops.common import _should_unpack
```

Comment:
We usually only import modules, so that it's easier to tell whether symbols are defined in the local file vs another module.

Reply:
addressed in the commit